include/usb/usb_device.h

```diff
@@ -433,6 +433,15 @@ bool usb_transfer_is_busy(uint8_t ep);
  */
 int usb_wakeup_request(void);
 
+/**
+ * @brief Get status of the USB remote wakeup feature
+ *
+ * Function to get status of remote wakeup feature.
```

Comment:
Is this description actually needed? It does not bring much more than the brief one.

Reply:
At first it was more useful, but the PR has changes a lot since last change in upstream. I will remove this.

Reply:
Done